### ISSUE TITLE
this.width() inconsistent reporting

### ISSUE THREAD

**Issue by nickedens** [jmoenig/Snap--Build-Your-Own-Blocks#1622]
Attached is the example file to reproduce the issue. I am working on an infinite scroll block for our students to use in an upcoming class. Everything works fine until you expand the stage view to full screen (with the full screen icon in Snap). When that happens the sprite which should be seemless ends up with gaps between copies. 

I narrowed this down to the javascript call `this.width() / 2` reporting inconsistent results. When in regular coding view (non full screen) you will see the sprite reports its size as 363 (half of its reported width). However when you run it full screen that exact same call reports its width as 544.5 even though that is not the width of the sprite. I ran into this when attempting to create an exe using the Snapp! website and then realized it happens even within Snap! itself. 

Any ideas? Is it possible I am just using the wrong javascript call?

[example.zip](https://github.com/jmoenig/Snap--Build-Your-Own-Blocks/files/709385/example.zip)

**Comment by jmoenig** [comment]
you gotta be careful when querying dimensions, because bounding boxes can be absolute and relative in some cases. The Morphic dimensions are those that are actually displayed on screen, and those vary as the whole microworld (stage) gets scaled, even thought the *logical* extent might stay the same

**Comment by nickedens** [comment]
Hmm. Ok so is there a different way to get just the dimensions of the sprite even when the world is scaled?

**Comment by jmoenig** [comment]
there is a scale property somewhere, I think the stage has it. (Sorry, I'm travelling...)

**Comment by nickedens** [comment]
@jmoenig I'll see if I can find it and post here if I can. If you happen to find it before I post please do let me know. Not a super hurry as my next class doesn't start until Feb.

**Comment by nickedens** [comment]
I found it and its all working properly now.

I ended up creating two separate blocks for it. Made it way easier (and more straight forward for the kids) to make the more complex custom blocks. 

![stage_scale](https://cloud.githubusercontent.com/assets/22965711/22004181/a59b1e50-dc27-11e6-85ff-ac4314687b87.png)

![sprite_width](https://cloud.githubusercontent.com/assets/22965711/22004198/c751fe9c-dc27-11e6-995a-eb2c49d1cf96.png)

Thank you again so much for your help!
